src/app/zap-templates/zcl/data-model/chip/service-area-cluster.xml

```diff
@@ -0,0 +1,131 @@
+<?xml version="1.0"?>
+<!--
+Copyright (c) 2023-24 Project CHIP Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<configurator>
+  <domain name="CHIP"/>
+
+  <bitmap name="Feature" type="bitmap32">
+    <cluster code="0x0150" />
+    <field name="ListOrder"           mask="0x01" />
+    <field name="SelectWhileRunning"  mask="0x02" />
+  </bitmap>
```

Comment:
There is a much richer way to define features - for instance

' <features>

     <features>
      <feature bit="0" code="DIRC" name="DirectCurrent" summary="Supports measurement of direct current">
        <optionalConform choice="a" more="true"/>
      </feature>
      <feature bit="1" code="ALTC" name="AlternatingCurrent" summary="Supports measurement of alternating current">
        <optionalConform choice="a" more="true"/>
      </feature>
      <feature bit="2" code="POLY" name="PolyphasePower" summary="Supports polyphase measurements">
        <optionalConform>
          <feature name="ALTC"/>
        </optionalConform>
      </feature>
      <feature bit="3" code="HARM" name="Harmonics" summary="Supports measurement of AC harmonics">
        <optionalConform>
          <feature name="ALTC"/>
        </optionalConform>
      </feature>
      <feature bit="4" code="PWRQ" name="PowerQuality" summary="Supports measurement of AC harmonic phases">
        <optionalConform>
          <feature name="ALTC"/>
        </optionalConform>
      </feature>
    </features>'

Reply:
@bzbarsky-apple is there a preference here? I agree that this does look better.